server/utils/readTime.util.js

```diff
@@ -0,0 +1,15 @@
+import readingTime from 'reading-time';
+import stripHtml from './stripHtml.util';
+import baseUtils from './articles/base.util';
+/**
+ * Returns reading time for a text
+ * @param {string} text
+ * @returns {string} reading time
+ */
+const readTime = async (text) => {
+  if (!baseUtils.isString(text)) return 'Content must be a string';
+  const plainText = await stripHtml(text);
```

Comment:
@geekdanitek Please, can you clarify why you used a library to get read time for an article, also does it cater for images?

Reply:
Currently, we don't cater for images

Reply:
@geekdanitek an article has at least one image, so it is necessary to cater to effect that change on the read time for reference check this [link](https://blog.medium.com/read-time-and-you-bc2048ab620c)

Reply:
@geekdanitek Please fix this pending issue